src/Raven.Client/Http/NodeSelector.cs

```diff
@@ -66,16 +66,19 @@ public bool OnUpdateTopology(Topology topology, bool forceUpdate = false)
             var state = _state;
             var stateFailures = state.Failures;
             var serverNodes = state.Nodes;
-            var len = Math.Min(serverNodes.Count, stateFailures.Length);
+            var numOfServers = serverNodes.Count;
+            var len = Math.Min(numOfServers, stateFailures.Length);
             for (var i = 0; i < len; i++)
```

Comment:
why it is saying unreachable code?

Reply:
I don't get this warning, Do you see it in the incremental phase of the loop?

Reply:
no longer